canonical-websites/www.ubuntu.com
# install vanilla-borchure-theme and fix up missing variables

## Done

Added vanilla-brochure-theme and fixed any broken variables.
note: these variables may be refactored or removed later, this PR is just to make the site run with the theme installed.

## QA

 - `./run` and see that there are no sass errors (the site will look very broken but this is the first step)
